### DISCUSSION TITLE
[不具合?] Pythonのシンタックスハイライトでシングル/ダブルクォーテーション内の三連ピリオドがシンタックスとして認識されてしまう

### DISCUSSION THREAD

**Discussion by osamusg** [increments/qiita-discussions#488] [要望・アイデア・不具合]
タイトル通りですが、要はMarkdownのコードブロック内で
print('...')
や
print("...")
と書くとクォーテーションが閉じないため、それ以下の行のリテラルとリテラル以外の認識が反転してしまう。
(以前はそんなことなかったような..)
print("'''")
や
print(',,,')
は問題なし。
Python以外の言語でどうなるかは調べていません。

**Comment by tanaka-asami415** [comment]
@osamusg 
ご報告ありがとうございます🙏
こちら修正いたします！

**Comment by getty104** [comment]
@osamusg 

お待たせしました、こちら修正したのでご確認お願いします！
![スクリーンショット 2023-08-07 11 44 45](https://github.com/increments/qiita-discussions/assets/20335778/ffdae239-0448-4008-99df-5b7606e3d98d)
